PATCH-P0-FEATURE-SUSE: Use better comments in install_db

We ship init script so we don't need to warn user to install one and we also
have a testsuite in separate packege.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: scripts/mysql_install_db.sh
===================================================================
--- scripts/mysql_install_db.sh.orig
+++ scripts/mysql_install_db.sh
@@ -521,9 +521,10 @@ fi
 # the screen.
 if test "$cross_bootstrap" -eq 0 && test -z "$srcdir"
 then
-  s_echo
-  s_echo "To start mysqld at boot time you have to copy"
-  s_echo "support-files/mysql.server to the right place for your system"
+# Output disabled, since the SUSE RPM comes with an init script installed
+#  s_echo
+#  s_echo "To start mysqld at boot time you have to copy"
+#  s_echo "support-files/mysql.server to the right place for your system"
 
   if test "$auth_root_authentication_method" = normal
   then
@@ -551,10 +552,9 @@ then
   then
     echo
     echo "You can start the MariaDB daemon with:"
-    echo "cd '$basedir' ; $bindir/mysqld_safe --datadir='$ldata'"
+    echo "rcmysql start"
     echo
-    echo "You can test the MariaDB daemon with mysql-test-run.pl"
-    echo "cd '$basedir/mysql-test' ; perl mysql-test-run.pl"
+    echo "You can test the MariaDB daemon with mariadb-test package"
   fi
 
   echo
